src/components/PlantsV2/components/LiveDeadPlantsPerSpeciesCard.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from 'react';
+import OverviewItemCard from 'src/components/common/OverviewItemCard';
+import strings from 'src/strings';
+import { Box, Typography, useTheme } from '@mui/material';
+import BarChart from 'src/components/common/Chart/BarChart';
+import { ObservationResults } from 'src/types/Observations';
+import { Dropdown } from '@terraware/web-components';
+
+type LiveDeadPlantsPerSpeciesCardProps = {
+  observation?: ObservationResults;
+};
+
+export default function LiveDeadPlantsPerSpeciesCard({ observation }: LiveDeadPlantsPerSpeciesCardProps): JSX.Element {
+  const theme = useTheme();
+  const [labels, setLabels] = useState<string[]>();
+  const [values, setValues] = useState<number[]>();
+  const [selectedSpecies, setSelectedSpecies] = useState<string>();
+  const [allSpecies, setAllSpecies] = useState<
+    {
+      label: string;
+      value: string;
+    }[]
+  >();
+  useEffect(() => {
+    if (observation) {
+      const speciesNames = observation.species.map((sp) => ({
+        label: sp.speciesScientificName || '',
+        value: sp.speciesId?.toString() || '',
+      }));
+      setAllSpecies(speciesNames);
+    }
+  }, [observation]);
+
+  useEffect(() => {
+    if (selectedSpecies) {
+      setLabels([strings.LIVE, strings.DEAD]);
+      const selectedObservationSpecies = observation?.species.find(
+        (sp) => sp.speciesId?.toString() === selectedSpecies
+      );
+      if (selectedObservationSpecies) {
+        const totalPlants = selectedObservationSpecies.totalPlants;
+        const dead = ((selectedObservationSpecies.mortalityRate || 0) * totalPlants) / 100;
```

Comment:
yeah.. i noticed we have some null mortality rates.. ideally have the redux framework set the moratlity rate to 0 in the reducer or selector.. all other components will get this adjustment for free